x/pairing/client/cli/tx_stake_provider.go

```diff
@@ -312,3 +361,26 @@ func getValidator(clientCtx client.Context, provider string) string {
 	}
 	return validatorBiggest.OperatorAddress
 }
+
+func parseOperator(clientCtx client.Context, operator string) (string, error) {
```

Comment:
this is a duplicate

Reply:
done. see https://github.com/lavanet/lava/commit/8fa736e4fc672043c1744feec22aa8bc6ebec4ed